networking/src/ApiClient.ts

```diff
@@ -843,33 +832,41 @@ class QuestionTemplateParser {
     static questionTemplateFromAWSQuestionTemplate(
         awsQuestionTemplate: AWSQuestionTemplate
     ): IQuestionTemplate {
+        let gameTemplates: IGameTemplate[] = [];
+        if (!isNullOrUndefined(awsQuestionTemplate) && !isNullOrUndefined(awsQuestionTemplate.gameTemplates) && !isNullOrUndefined(awsQuestionTemplate.gameTemplates.items)) {
```

Comment:
We should break this into multiple lines for readability

Reply:
fixed